### PR TITLE
add method updateChatContext

### PR DESCRIPTION
**Pull request by shubhamtewari**

### CODE REVIEW

File: kommunicate/src/main/java/io/kommunicate/Kommunicate.java [hunk 1]
```diff
@@ -30,6 +31,11 @@
 import com.applozic.mobicommons.commons.core.utils.Utils;
 import com.applozic.mobicommons.json.GsonUtils;
 import com.applozic.mobicommons.people.channel.Channel;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.reflect.TypeToken;
+
+import org.json.JSONObject;
```
Comment (devashishmamgain) [inline kommunicate/src/main/java/io/kommunicate/Kommunicate.java]: @shubhamtewari is this getting used?

Comment (devashishmamgain) [reply kommunicate/src/main/java/io/kommunicate/Kommunicate.java]: @reytum is we are using Gson then do we need to use org.json.JSONObject as well?

**Comment by devashishmamgain** [comment]
@shubhamtewari what is expected from this method:
lets say if there is already one key value pair stored as:

sessionId: abc

and new key value pair comes as:
language: spanish

then what is the expected final value:

session: abc
language: spanish

or just
language: spanish

File: kommunicate/src/main/java/io/kommunicate/Kommunicate.java (hunk 1 above)
Comment (shubhamtewari) [reply kommunicate/src/main/java/io/kommunicate/Kommunicate.java]: we dont require the import, removed it

**Comment by shubhamtewari** [comment]
> @shubhamtewari what is expected from this method:
> lets say if there is already one key value pair stored as:
> 
> sessionId: abc
> 
> and new key value pair comes as:
> language: spanish
> 
> then what is the expected final value:
> 
> session: abc
> language: spanish
> 
> or just
> language: spanish

it the data previously stored in the KmChatContext key will be updated
session: abc
language: spanish

**Pull request closed without merging**